airbyte_cdk/sources/declarative/retrievers/simple_retriever.py

```diff
@@ -670,6 +670,7 @@ def _fetch_next_page(
         stream_slice: StreamSlice,
         next_page_token: Optional[Mapping[str, Any]] = None,
     ) -> Optional[requests.Response]:
+        is_dynamic_schema_call = self.name == "dynamic_properties"
```

Comment:
I'm not super in love with this solution because someone could definitely change the name of the retriever without thinking it would break something in the connector builder. It also feels like the log formatter is still wrong as it would return Could we `f"Request performed in order to extract records for stream '{self.name}'"` as a description. Could we make the log_formatter a parameter of the SimpleRetriever as a nullable parameter and remove the override from `SimpleRetrieverTestReadDecorator`?